app/controllers/admin/dashboard_controller.rb

```diff
@@ -4,43 +4,55 @@ def show
 
     Groupdate.time_zone = "New Delhi"
 
-    @users_requesting_approval = User.requested_sync_approval
-
-    @facilities = Facility.all.order(:name)
+    @users_requesting_approval = User.where(facility: current_admin.facility_groups.flat_map(&:facilities)).requested_sync_approval
```

Comment:
will requested_sync_approval work here?
it is a class method on User, but here it's being called on an array.